src/utils/polygon.h

```diff
@@ -546,9 +546,16 @@ class Polygon : public PolygonRef
     {
     }
 
-    Polygon& operator=(const ConstPolygonRef& other)
+    Polygon& operator=(const ConstPolygonRef& other) = delete; // copying a single polygon is generally not what you want
+//     {
+//         path = other.path;
+//         poly = *other.path;
+//         return *this;
+//     }
+
+    Polygon& operator=(Polygon&& other) //!< move assignment
     {
-        path = other.path;
+        poly = std::move(other.poly);
         return *this;
     }
```

Comment:
Implementation in header.

Reply:
I barely touched the class!
Uhm. changing polygon.h is scary!

I think that if we want to clean up this class we should do it all at once. Let's not do that as part of this PR.